python/requirements.txt

```diff
@@ -1,6 +1,6 @@
-openai==0.27.*
-numpy==1.24.*
-aiofiles>=23.1.0
-transformers>=4.28.0
-sentence-transformers>=2.2.2
-torch>=2.0.0
\ No newline at end of file
+openai==0.27.0
+numpy==1.24.2
+aiofiles==23.1.0
+transformers==4.28.0
+sentence-transformers==2.2.2
+torch==2.0.0
```

Comment:
change needed for the tests? could you include this (motivation/description) in the PR?

Reply:
I pinned the requirements to the same versions pinned in the pyrpoject.toml. With the flexible versioning here, there is risk that our local dependencies are out of sync with what we publish as a pip package. This could, in theory, affect behavior and testing. 

see: 


```
[tool.poetry.dependencies]
python = "^3.8"
numpy = "^1.24.2"
openai = "^0.27.0"
aiofiles = "^23.1.0"
transformers = "^4.28.1"
torch = "^2.0.0"
sentence-transformers = "^2.2.2"
```